defaults/main.yml

```diff
@@ -605,7 +631,7 @@ postgresql_autovacuum_freeze_max_age: 200000000
 # maximum Multixact age before forced vacuum (>= 9.3)
 postgresql_autovacuum_multixact_freeze_max_age: 400000000
 # default vacuum cost delay for autovacuum, in milliseconds
-postgresql_autovacuum_vacuum_cost_delay: 20ms
+postgresql_autovacuum_vacuum_cost_delay: 2ms     # (<= 11: 20ms, >=12 2ms)
```

Comment:
New default for v12

Reply:
The old default version is now set in vars/postgresql_*.yml... so behaviour shouldn't change for old versions.